### PR TITLE
Breaking up Go Signal content

### PR DESCRIPTION
**Pull request by flossypurse**

### CODE REVIEW

File: docs/go/how-to-handle-a-signal-in-a-workflow-in-go.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
+---
+id: how-to-handle-a-signal-in-a-workflow-in-go
+title: How to handle a Signal in a Workflow in Go
+sidebar_label: Handling Signals
+description: TODO
```
Comment (flossypurse) [inline docs/go/how-to-handle-a-signal-in-a-workflow-in-go.md:5]:
```suggestion
description: Use the `GetSignalChannel()` API from the `go.temporal.io/sdk/workflow` package to get the Signal Channel.
```

File: docs/go/how-to-send-a-signal-to-a-workflow-execution-in-go.md [hunk 2]
```diff
@@ -0,0 +1,83 @@
+---
+id: how-to-send-a-signal-to-a-workflow-execution-in-go
+title: How to send a Signal to a Workflow Execution in Go
+sidebar_label: Sending Signals
+description: todo
```
Comment (flossypurse) [inline docs/go/how-to-send-a-signal-to-a-workflow-execution-in-go.md:5]:
```suggestion
description: Use the `SignalWorkflow()` method on and instance of the Go SDK Temporal Client to send a Signal to a Workflow Execution.
```

**Comment by cretz** [comment]
Question - why are we breaking up the Go docs into smaller and smaller pages? The idea of just going to https://docs.temporal.io/docs/go/signals/ and reading everything about signals (or https://docs.temporal.io/docs/go/workflows/ and reading everything about workflows, etc) is arguably much more approachable than the growing number of disparate pages.

File: docs/go/how-to-send-a-signal-to-a-workflow-execution-in-go.md [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+id: how-to-send-a-signal-to-a-workflow-execution-in-go
+title: How to send a Signal to a Workflow Execution in Go
+sidebar_label: Sending Signals
+description: Use the `SignalWorkflow()` method on and instance of the Go SDK Temporal Client to send a Signal to a Workflow Execution.
+tags:
+  - go
+  - developer-guide
+---
+
+Use the `SignalWorkflow()` method on and instance of the [Go SDK Temporal Client](https://pkg.go.dev/go.temporal.io/sdk/client#Client) to send a [Signal](/docs/content/what-is-a-signal) to a [Workflow Execution](/docs/content/what-is-a-workflow-execution).
```
Comment (aarohib) [inline docs/go/how-to-send-a-signal-to-a-workflow-execution-in-go.md:11]: "Use the `SignalWorkflow()` method on **_an_** instance of.."

File: docs/go/how-to-send-a-signal-to-a-workflow-execution-in-go.md [hunk 4]
```diff
@@ -0,0 +1,83 @@
...
+	log.Fatalln("Error sending the Signal", err)
+	return
+}
+```
+
+### Signal-With-Start
+
+If you are unsure of the status of the Workflow Execution that you are sending the Signal to, you can use the `SignalWithStartWorkflow()` method on the Go SDK Temporal Client to start the Workflow Execution and give it the Signal it at the same time.
+
+This method does not take a Run Id as a parameter.
```
Comment (aarohib) [inline docs/go/how-to-send-a-signal-to-a-workflow-execution-in-go.md:61]: "This method does not take Run ID as a parameter." or "This method does not require a Run ID parameter."?

File: docs/go/how-to-send-a-signal-to-a-workflow-execution-in-go.md [hunk 5]
```diff
@@ -0,0 +1,83 @@
...
+err = temporalClient.SignalWithStartWorkflow(context.Background(), "your-workflow-id", "your-signal-name", signal)
+if err != nil {
+	log.Fatalln("Error sending the Signal", err)
+	return
+}
+```
+
+### How to send a Signal from within a Workflow
+
+A Workflow can send a Signal to another Workflow Execution using the [`SignalExternalWorkflow`](https://pkg.go.dev/go.temporal.io/sdk/workflow#SignalExternalWorkflow) API from the `go.temporal.io/sdk/workflow` package.
```
Comment (aarohib) [inline docs/go/how-to-send-a-signal-to-a-workflow-execution-in-go.md:74]: "A Workflow can send a Signal to a different Workflow Execution using..." ? Or "You can send a signal from within a Workflow to a different Workflow Execution using ..." > Wordier, but ties in the header more clearly.

**Review by aarohib — APPROVED** [review]

**Pull request merged**